### ISSUE TITLE
Somehow broken muxed ass works in player but throw error while parsing

### ISSUE THREAD

**Issue by bigretromike** [chireiden/python-ass#6]
error:
`ValueError: invalid literal for int() with base 10: 'Comment'`

Like you see the `Marked=0` is in wrong order, while also missing `Format`
Somehow this work on most players I tried, ffmpeg extract it this way while its not ffmpeg bug.
I'm unable to parse it in python-ass so I could fix it via re-dumping. 

body
```
[Script Info]
; Script generated by Aegisub v1.10 
; http://www.aegisub.net
Title: <untitled>
Original Script: <unknown>
ScriptType: v4.00+
Collisions: Normal
PlayResY: 1024
PlayDepth: 0
Timer: 100.0000
WrapStyle: 0
Last Style Storage: Default
Video Aspect Ratio: 0
Video Zoom: 6
Video Position: 0

[V4+ Styles]
Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outli
ne, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
Style: Style1,Candara,70,&H00FFFFFF,&H0080FFFF,&H00306000,&H80306000,0,0,0,0,100,100,0,0,1,3,2,2,30,30,25,0
Style: Style2,Candara,60,&H00FFFFFF,&H0080FFFF,&H00603000,&H80603000,0,0,0,0,100,100,0,0,1,3,2,8,30,30,25,0

[Events]
Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
Dialogue: 0,0:01:48.70,0:01:50.23,Marked=0,Style1,Comment,0000,0000,0000,,Pathetic.
Dialogue: 0,0:01:50.31,0:01:54.11,Marked=0,Style1,Comment,0000,0000,0000,,Can't ask much from a beginner\nwho just joined the game.
Dialogue: 0,0:02:11.36,0:02:14.45,Marked=0,Style1,Comment,0000,0000,0000,,That was a disaster.\NYou'd better be careful.
Dialogue: 0,0:02:15.03,0:02:18.02,Marked=0,Style1,Comment,0000,0000,0000,,Right now, there are\nmany PK's in "The World".
Dialogue: 0,0:02:19.67,0:02:20.43,Marked=0,Style1,Comment,0000,0000,0000,,PK?
Dialogue: 0,0:02:21.40,0:02:23.06,Marked=0,Style1,Comment,0000,0000,0000,,Player Killers.
Dialogue: 0,0:02:23.14,0:02:24.94,Marked=0,Style1,Comment,0000,0000,0000,,They hunt and kill other players.
Dialogue: 0,0:02:26.01,0:02:27.70,Marked=0,Style1,Comment,0000,0000,0000,,Like those guys before?
```

For now I'm removing `Marked=0` which helps a lot, because Adding `Marked` in Format before `Name` does not help

**Comment by FichteFoll** [comment]
That file definitely looks illegal. As you mentioned, `Marked=0` has no entry in the Format line and I have never seen this in a ASS 4+ formated file either. I don't see how we should parse this.

**Comment by bigretromike** [comment]
yet, it works flawless in most if not all players.

**Comment by FichteFoll** [comment]
If anyone could point me to actual parsing rules for this kind of file (or how ffmpeg/players parse it), I would be willing to look into it.

**Comment by nitori** [comment]
I took a quick look at it, and it doesn't seem to work correctly on MPV and VLC players.

In both cases the last `,` ends up being part of the displayed text (because we got one field value too much and it treats everything after the last valid `,` as part of the Text field), and in both cases the style is completely ignored.

Usually it would be displayed as big white text if the "Marked=0" field is removed.

It seems the Marked=0 part is something from SubStation Alpha (ssa), and usually sits on the Layer position at first place. Though in that case "Layer" would named "Marked" instead.

https://en.wikipedia.org/wiki/SubStation_Alpha#Principal_sections_in_an_SSA_subtitle

What players did you try?

![2021-03-27_171807](https://user-images.githubusercontent.com/1561831/112726980-73d50d80-8f20-11eb-87a4-f3e33d794dc2.png)
![2021-03-27_172051](https://user-images.githubusercontent.com/1561831/112727053-d0382d00-8f20-11eb-969a-c69ec9ba1df8.png)
